addons/survey/views/survey_templates_print.xml

```diff
@@ -46,6 +46,8 @@
                                     </div>
                                 </t>
                             </t>
+                            <!--Set question to false to avoid background miss match as last question is still in context and is used in survey.layout t-call.-->
+                            <t t-set="question" t-value="False"/>
```

Comment:
I think we talked about renaming that variable, as we had issues with it in stable... something like print_question, result_question, ... ?